City-of-Helsinki/drupal-hdbt
# UHF-4532: Small adjustments to the daycare search styling

# Daycare search styling changes [UHF-4532](https://helsinkisolutionoffice.atlassian.net/browse/UHF-4532)
Small adjustments to the daycare search styling

## What was done
* Adjusted distance information position on the unit__minimal
* Changed the margin to correct between unit cards on daycare search
* Changed the distance sort alert to have styles from hds-notification type alert.

## How to install
* Make sure your KASKO instance is up and running on latest dev branch.
    * `git pull origin dev`
    * `make fresh`
* Update the HDBT theme
    * `composer require drupal/hdbt:dev-UHF-4532_daycare_search_style_adjustments`
* Update the helfi_tpr module
    * `composer require drupal/helfi_tpr:dev-UHF-4532_daycare_search_style_adjustments`
* Run `make drush-cr`

## How to test
* Go to daycare search
    * If you don't have one you need to add daycare taxonomy term to the unit type taxonomy
    * Make sure the daycare search view uses the term you just added
    * Tag the daycare term to some TPR units
    * Go to basic page and add the daycare search to lower content.
* Now check that the margin between the cards is 8px.
* Check that if you type a correct address that the distance to that address is aligned close to the arrow on the unit card.
* Write an address that is not correct and you should see a notification that is similar to the screenshot.

## Other PRs
* https://github.com/City-of-Helsinki/drupal-module-helfi-tpr/pull/112
* https://github.com/City-of-Helsinki/drupal-helfi-kasvatus-koulutus/pull/129
